src/store/getters.ts

```diff
@@ -182,6 +199,14 @@ export const isRestoration = (state: StateIF): boolean => {
   return (getRequestActionCd(state) === NrRequestActionCodes.RESTORE)
 }
 
+/** True if current flow is XPRO (includes AMALGAMATION, NEW_BUSINESS, and possibly others). */
+export const getIsXproFlow = (state: StateIF): boolean => {
+  return (
+    (getRequestActionCd(state) !== NrRequestActionCodes.MOVE) &&
+    [Location.CA, Location.IN].includes(getLocation(state))
+  )
+}
```

Comment:
Note that "MOVE" from Canada or International doesn't seem to count as a XPRO flow 🤷‍♂️